sim/core/aura.go

```diff
@@ -203,14 +204,16 @@ type auraTracker struct {
 
 	aurasByTag map[string][]*Aura
 
-	// IDs of Auras that are active, in no particular order.
+	// IDs of Auras that may expire and are currently active, in no particular order.
 	activeAuras []*Aura
 
+	// caches the minimum expires time of all active auras; reset to 0 on Activate(), Deactivate(), and Refresh()
+	minExpires time.Duration
+
 	// Auras that have a non-nil XXX function set and are currently active.
 	onCastCompleteAuras   []*Aura
 	onSpellHitAuras       []*Aura
 	onPeriodicDamageAuras []*Aura
-	onMeleeAttackAuras    []*Aura
```

Comment:
Unused, as far as I can see.

Reply:
I know this used to be used but there has been a lot of aura refactoring recently so it might be now.